R/endpoints.R

```diff
@@ -1,16 +1,20 @@
-#' fetch AFHSB data (point data, no min/max)
+#' Fetch AFHSB data (point data, no min/max)
 #'
-#' API docs: https://cmu-delphi.github.io/delphi-epidata/api/afhsb.html
+#' @param auth Character. Authentication token.
+#' @param locations Character vector. Locations to fetch (region/state, or 3-letter country code labels).
+#' @param epiweeks Epirange. Epiweeks to fetch in the form epirange(startweek, endweek), where
+#'   startweek and endweek are in the form YYYYWW.
+#' @param flu_types Character. Flu types to fetch (disjoint (flu1, flu2-flu1, flu3-flu2, ili-flu3)
+#'   or subset (flu2, flu3, ili) flu type labels).
+#' @return An instance of epidata_call.
 #'
-#' @param auth character authentication token
-#' @param locations character locations to fetch
-#' @param epiweeks epirange epiweeks to fetch
-#' @param flu_types character flu_types to fetch
-#' @return an instance of epidata_call
+#' @references API docs: https://cmu-delphi.github.io/delphi-epidata/api/afhsb.html
```

Comment:
quibble: References appear at the bottom in pkgdown, but these API doc references are the only place where we actually describe the nature of the data set in more than a sentence (well, maybe not for this particular endpoint, but hopefully for others).

Reply:
Should we not use the @references tag then and just move them back up under the header?